protostar/commands/test/environments/fuzz_test_execution_environment.py

```diff
@@ -258,3 +275,24 @@ class RunsCounter:
     def __next__(self) -> int:
         self.count += 1
         return self.count
+
+
+class FuzzTestCaseCheatcodeFactory(TestCaseCheatcodeFactory):
+    def __init__(
+        self,
+        state: TestExecutionState,
+        expect_revert_context: ExpectRevertContext,
+        finish_hook: Hook,
+    ):
+        super().__init__(state, expect_revert_context, finish_hook)
```

Comment:
You could pass `TestCaseCheatcodyFactory` in the constructor. By doing so, we don't have to update `FuzzTestCaseCheatcodeFactory` everytime `TestCaseCheatcodeFactory` changes.

This approach breaks the following rules:
- Open/Closed principle
- composition over inheritance
- DRY

Reply:
This is also a useless constructor. You can remove it to temporarily fix the DRY problem.

Refactor can be done in a different PR.